kube_scheduler/assets/dashboards/overview.json

```diff
@@ -1,642 +1,988 @@
-    {
-      "id": 19,
-      "definition": {
-        "type": "query_value",
-        "requests": [
-          {
-            "q": "max:kube_scheduler.goroutines{$cluster_name,$scope}",
-            "aggregator": "last"
-          }
-        ],
-        "custom_links": [],
-        "title": "Goroutines",
-        "title_size": "16",
-        "title_align": "left",
-        "autoscale": true,
-        "precision": 0
-      },
-      "layout": {
-        "x": 3,
-        "y": 88,
-        "width": 15,
-        "height": 9
-      }
-    },
-    {
-      "id": 20,
-      "definition": {
-        "type": "query_value",
-        "requests": [
-          {
-            "q": "max:kube_scheduler.threads{$cluster_name,$scope}",
-            "aggregator": "last"
-          }
-        ],
-        "custom_links": [],
-        "title": "Threads",
-        "title_size": "16",
-        "title_align": "left",
-        "autoscale": true,
-        "precision": 0
-      },
-      "layout": {
-        "x": 3,
-        "y": 66,
-        "width": 15,
-        "height": 9
-      }
-    },
-    {
-      "id": 21,
-      "definition": {
-        "type": "query_value",
-        "requests": [
-          {
-            "q": "max:kube_scheduler.open_fds{$cluster_name,$scope}",
-            "aggregator": "last"
-          }
-        ],
-        "custom_links": [],
-        "title": "Open FDS",
-        "title_size": "16",
-        "title_align": "left",
-        "autoscale": true,
-        "precision": 0
-      },
-      "layout": {
-        "x": 3,
-        "y": 77,
-        "width": 15,
-        "height": 9
-      }
-    }
-  ],
-  "template_variables": [
-    {
-      "name": "scope",
-      "default": "*",
-      "prefix": null
-    },
-    {
-      "name": "cluster_name",
-      "default": "*",
-      "prefix": "kube_cluster_name"
-    }
-  ],
-  "layout_type": "free",
-  "is_read_only": true,
-  "notify_list": [],
-  "id": 30270
+	"title": "Kubernetes Scheduler Overview",
+	"description": "The Kubernetes scheduler is a control plane process which assigns Pods to Nodes. This makes it a critical component in the Kubernetes control plane that needs to be monitored.\n\nThe Datadog Kubernetes Scheduler Overview dashboard gives you broad visibility into the status of the scheduler, including the state of its scheduling queues and preempted pods.\n\nFurther reading around the Kubernetes Scheduler:\n\n- [Datadog Kubernetes Scheduler integration documentation](https://docs.datadoghq.com/integrations/kube_scheduler/)\n\n- [Kubernetes Scheduler official documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/)\n\n- [Monitoring in the Kubernetes era](https://www.datadoghq.com/blog/monitoring-kubernetes-era/)\n\nCheck [the agent documentation](https://docs.datadoghq.com/agent/kubernetes/) if some of the graphs are empty.\nClone this template dashboard to make changes and add your own graph widgets. ",
+	"widgets": [
+		{
+			"id": 895563546267612,
```

Comment:
I'm wondering if `id` should be present.

Reply:
Not sure if this is required. Previous dashboards I contributed to also had these ids.